src/main/java/com/example/demo/matcher/Matcher.java

```diff
@@ -43,7 +40,7 @@ public void match(Order newOrder) {
                     sellOrder.getOrderId(),
                     tradePrice,
                     tradeQuantity,
-                    LocalDateTime.now()));
+                    Instant.now().toEpochMilli()));
```

Comment:
By default when a DateTime is serialized, it gets serialized to a string. So when I send an Order over the API to the client, the client will then have to parse the string into its own implementation of datetime. It's much simpler to just deal with the datetimes as an integer number of millis since the epoch, because that is standard across the client and server. (plus that's how JS serialises datetimes)